### DISCUSSION TITLE
Can't remove "content/docs" from "Edit this page" URL

### DISCUSSION THREAD

**Discussion by mojavezax** [google/docsy#2138] [Q&A]
Hello, I am using Hugo v0.138.0 and Docsy 0.11.0.
I'm having a problem getting the right URL for the "Edit this page" link. I have a Hugo repository called `hugo`, and my content is in a separate repo called `notes`. The latter is a submodule mounted at `hugo/content/docs`. I have these config parameters in my `hugo/config/_default/hugo.toml` file:
```
github_repo = "https://github.com/mycompany/notes"
github_subdir = ""
path_base_for_github_subdir = ""
github_branch = "master"
```
So for a document in the `notes` repo called `stuff/doc.md`, I expect the URL to be
https://github.com/mycompany/notes/edit/master/stuff/doc.md, but instead I'm getting
https://github.com/mycompany/notes/edit/master/content/docs/stuff/doc.md
I've tried the above parameters in the top level `_index.md` as well as `stuff/_index.md`, but there's no difference. How can I avoid the `content/docs` in the URL?
If the above parameters won't do what I want then I don't mind hacking the theme, but I don't know where the URL is constructed.

**Comment by fekete-robert** [comment]
Hi, the actual URL is created in the themes/docsy/layouts/partials/page-meta-links.html file. I think you can also set it in the front matter of a page, but haven't had the time to check it. If it works, you can use cascade to set it for the entire subtree.

**Reply by mojavezax** [reply]
Thanks, I see it there. Maybe the cause is in this line:
`{{ $path := strings.TrimPrefix (add hugo.WorkingDir "/") $.File.Filename -}}`. Perhaps `hugo.WorkingDir` is set to `content/docs`?
I've tried setting those paramters in the front matter too, but it doesn't change anything.

**Comment by mojavezax** [comment]
I found an ugly hack that works for my site. In the file above I changed
`{{ $path := strings.TrimPrefix (add hugo.WorkingDir "/") $.File.Filename -}}`
to
`{{ $path := strings.TrimPrefix (add hugo.WorkingDir "/content/docs/") $.File.Filename -}}`
